lib/Onnxifi/HostManagerOnnxifi.cpp

```diff
@@ -27,6 +27,7 @@ bool GlowDumpDebugTraces = false;
 bool GlowSaturateHost = false;
 bool GlowFP16 = false;
 bool GlowClipFP16 = false;
+bool GlowDumpGraph = false;
```

Comment:
hm, I'm not sure we need to have it here now (once it's used by some open sourced backend we could add that).